source/PreferencesPanel.cpp

```diff
@@ -813,6 +867,7 @@ void PreferencesPanel::DrawPlugins()
 	const Color &dim = *GameData::Colors().Get("dim");
 	const Color &medium = *GameData::Colors().Get("medium");
 	const Color &bright = *GameData::Colors().Get("bright");
+	const Color &removed = *GameData::Colors().Get("plugin removed");
```

Comment:
Maybe add a color for plugins that have been updated?

Reply:
suggestion for a color?
a blue?

Reply:
Blue doesn't really fit, maybe some kind of yellow? idk

Reply:
yellow is already outdated